### ISSUE TITLE
Display Issue: Application Appears Out of Screen on Secondary Display

### ISSUE THREAD

**Issue by Skye119** [hugoattal/milky-warp#8]
It's perfect to use on my main display, but when I try to use it on the other display, it seems to be out of my screen. I can still see it in the taskbar preview,and the functionality is normal within the preview. It seems like the window position is not correct on secondary display.

**Comment by hugoattal** [comment]
Yes, that's by design. You need to trigger the zoom when your mouse in on the other monitor to zoom on that monitor.

Would you like some options to make it seamless for multiscreen setup? Though, I'm afraid there might be some weird things happening if your screens don't have the same pixel ratio.

**Comment by Skye119** [comment]
The issue occurs when I attempt to zoom in on a display while the mouse is on that screen which is not the main one. The window appears to go off the screen, not visible on either display.

**Comment by Skye119** [comment]
![Uploading abe9e6b09809bcaf82c5d8e838ac5f2.png…]()

**Comment by Skye119** [comment]
![abe9e6b09809bcaf82c5d8e838ac5f2](https://github.com/hugoattal/milky-warp/assets/73821415/84dbf19b-7176-44b6-9c90-38f61083d051)

**Comment by hugoattal** [comment]
That's weird... Are you using the last 1.3.0 version? https://github.com/hugoattal/milky-warp/releases
I did not implement auto-update yet, so you have to download the new version

(Nice wallpaper btw 😄 !)

**Comment by Skye119** [comment]
Yes, I am using the latest version. I test it, when I switch the main display in windows settings, it would occur on the other display 😭 So I simply changed the main display in order to use it. It's  helpful to see some small text（codes） in videos. Nice work.
